### PR TITLE
Check for a valid plugin context before accessing plugin context vari…

### PR DESCRIPTION
**Pull request by tdenewiler**
…ables related to showing tool output.

### CODE REVIEW

File: statick_tool/plugins/tool/cccc_tool_plugin.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def scan(  # pylint: disable=too-many-branches
                 print("Couldn't find cccc executable! ({})".format(ex))
                 return None
 
-            if self.plugin_context.args.show_tool_output:
+            if self.plugin_context and self.plugin_context.args.show_tool_output:
```
Comment (xydesa) [inline statick_tool/plugins/tool/cccc_tool_plugin.py:80]: this tool checks for self.plugin_context is None at the top of this function. is this change still necessary?

Comment (tdenewiler) [reply statick_tool/plugins/tool/cccc_tool_plugin.py:80]: Statick can still crash if `self.plugin_context` is empty. I've seen that happen a few times in the past.

**Review by xydesa — APPROVED** [review]
